test/integration/targets/azure_rm_virtualmachine/os_disk_size.yaml

```diff
@@ -0,0 +1,46 @@
+---
+
+- name: Create virtual machine with specific OS disk size
```

Comment:
It's too expensive to create a vm for each of test scenario, since vm creation/deletion is time  consuming, and there's  core limitation in test subscription. Suggest to combine some of those test, eg. image  disk  size with no_public ip test together.

Reply:
The motivation here is to increase the number of possible tests without timeout. AS `devel` stands currently we are at the cusp of timeouts with the number of tests.
 This module needs many more tests and what is present in this PR is still a somewhat minimal set.
Where possible VM's have been preserved, yet testing with managed disk, with specified OS disk size, and market place images require new VM's.
Essentially each _line_ of VM's is separated in the `inventory`.